CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
-cmake_minimum_required(VERSION 3.16)
+cmake_minimum_required(VERSION 3.22)
 project(ArborX CXX)
 
 if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.30.0)
   message(STATUS "Setting policy CMP0167 to use FindBoost module")
-  cmake_policy(SET CMP0167 OLD)
+  cmake_policy(SET CMP0167 NEW)
```

Comment:
I don't quite get that change.  Please explain.

Reply:
Admittedly, drive-by change. Instead of retaining old behavior, switch to the new one for finding boost modules. This allows to test both ways.

Reply:
I think the right way to do this is unconditionally setting the policy to new.
That is
```diff
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,10 +1,8 @@
 cmake_minimum_required(VERSION 3.16)
 project(ArborX CXX)

-if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.30.0)
-  message(STATUS "Setting policy CMP0167 to use FindBoost module")
-  cmake_policy(SET CMP0167 OLD)
-endif()
+# FindBoost module removed in version 3.30
+cmake_policy(SET CMP0167 OLD)

 # use gnu standard install directories
 include(GNUInstallDirs)

Reply:
If you do that, you will get warnings for older versions of CMake
```
Policy "CMP0167" is not known to this version of CMake.
```